src/App.js

```diff
@@ -1,14 +1,36 @@
 import {Navbar} from "./components/navbar/Navbar";
 import {LandingPage} from "./components/LandingPage/LandingPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import {JitsiComponent} from "e-stella-jitsi";
 
 
 function App() {
   return (
     <div>
       <Navbar />
-      <div style={{marginTop: "2em"}}>
-          <LandingPage />
-            <a href='https://www.freepik.com/vectors/people' style={{display:"none"}}>People vector created by pikisuperstar - www.freepik.com</a>
+      <div>
+          <Router>
+              <Switch>
+
+                  <Route exact path="/">
+                      <div style={{marginTop: "2em"}}>
+                          <LandingPage />
+                          <a href='https://www.freepik.com/vectors/people' style={{display:"none"}}>People vector created by pikisuperstar - www.freepik.com</a>
```

Comment:
Yyyy na pewno czegoś takiego potrzebujemy?

Reply:
Jeśli chcemy używać, to powinniśmy, żeby być zgodni z prawem. Zobacz, że potem jest display: none, czyli tego nie widać, ale jest